.github/workflows/dispatch.yaml

```diff
@@ -0,0 +1,16 @@
+name: Simple workflow with dispatch trigger
+
+on:
+  workflow_dispatch:
+    inputs:
+      string_parameter:
+        description: 'Simple simple parameter'
```

Comment:
nit: I guess a single "simple" would make this "simpler" 😁

Reply:
😂 damn, such a small file and with typos